website/source/security.html.erb

```diff
@@ -0,0 +1,32 @@
+---
+layout: "inner"
+page_title: "Security"
+description: |-
+  Terraform takes security very seriously. Please responsibly disclose any security vulnerabilities found and we'll handle it quickly.
+---
+
+<h1>Terraform Security</h1>
+
+<p>
+  We understand that many users place a high level of trust in HashiCorp
+  and the tools we build. We apply best practices and focus on security to
+  make sure we can maintain the trust of the community.
```

Comment:
This statement is kinda empty? Perhaps list some practices? We:

* We code review all changes for security concerns.
* We use and require encryption when possible.
* We publish secure packages and checksums for binaries.
* We run a security bounty program...

And other stuff that is relevant.

Or link to a central page that lists central Hashicorp security practice...